lienflash/lienflash-fe
# Add logo and basic styling to header

#### What’s this PR do?
- Add lienflash logo to app
- create basic styling for header

#### Where should the reviewer start?
- Header file for changes
#### How should this be manually tested?
- Once previous PR is merged, merge and in terminal run npm start to see it displayed

#### What are the relevant tickets?
#23 

#### Need Help/Support
- Need to figure out conditional logic of page
